charts/ccd-data-store-api/values.preview.template.yaml

```diff
@@ -15,8 +15,8 @@ java:
     DATA_STORE_DB_PASSWORD: "{{ .Values.postgresql.postgresqlPassword}}"
     DATA_STORE_DB_OPTIONS: "?stringtype=unspecified"
     DATA_STORE_DB_MAX_POOL_SIZE: 10
-    DEFINITION_STORE_HOST: http://ccd-definition-store-api-pr-575.service.core-compute-preview.internal/
-    USER_PROFILE_HOST: http://ccd-user-profile-api-pr-224.service.core-compute-preview.internal/
+    #DEFINITION_STORE_HOST: http://ccd-definition-store-api-pr-575.service.core-compute-preview.internal/
```

Comment:
Any reason to comment these? Do we want o keep aat urls on master branch for preview?

Reply:
Yes, we need to comment these to be able to wire the pods in preview with ccd dependencies on aat.
We will change it back as soon as we merge back to develop from master, after release.

Reply:
The pipeline improvement is taking care of wiring for FTAs on running on Jenkins agents, not those of CI/CD releasables on preview. That part requires further work to make more practically manageable.